gamestonk_terminal/technical_analysis/ta_controller.py

```diff
@@ -114,6 +116,8 @@ def print_help(self):
         print("volume:")
         print("   ad          chaikin accumulation/distribution line values")
         print("   obv         on balance volume")
+        print("custom")
+        print("   fib          fibinocci retracement")
```

Comment:
`custom` missing the `:`

`fibonacci retracement` has an extra space at the start.